### PR TITLE
created_project_table

### PR DESCRIPTION
**Pull request by pg20**

### CODE REVIEW

File: db/migrate/20151221061334_create_projects_table.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateProjectsTable < ActiveRecord::Migration
+  def change
+    create_table :projects_tables do |t|
+      t.string :project_title
+      t.string :language
+      t.string :description
```
Comment (DMips) [inline db/migrate/20151221061334_create_projects_table.rb]: let's change it to type text as string allows only 255 characters

File: db/migrate/20151221061334_create_projects_table.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateProjectsTable < ActiveRecord::Migration
+  def change
+    create_table :projects_tables do |t|
+      t.string :project_title
```
Comment (DMips) [inline db/migrate/20151221061334_create_projects_table.rb]: let's just call it title

File: db/migrate/20151221061334_create_projects_table.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateProjectsTable < ActiveRecord::Migration
+  def change
+    create_table :projects_tables do |t|
```
Comment (DMips) [inline db/migrate/20151221061334_create_projects_table.rb]: just projects is enough :)

File: app/views/mentee_profiles/dashboard.html.slim [hunk 4]
```diff
@@ -3,12 +3,11 @@
...
-    | Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-    | tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-    | quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-    | consequat.
+   = @project.language
+   br
+   = @project.description 
   a href="#"  Read more
   .git-link
     a href=""
```
Comment (pg20) [inline app/views/mentee_profiles/dashboard.html.slim]: where will the links here will redirect to ?

File: app/views/mentee_profiles/dashboard.html.slim [hunk 5]
```diff
@@ -18,7 +17,7 @@
   .progress
     .progress-bar aria-valuemax="100" aria-valuemin="0" aria-valuenow="60" role="progressbar" style=("width: 60%;") 
       | 60%
-  a href="#"  See entire project
+  = link_to  "See entire project", mentee_profile_url(current_user.id)
```
Comment (pg20) [inline app/views/mentee_profiles/dashboard.html.slim]: redirected to show page

File: app/views/mentee_profiles/show.html.slim [hunk 6]
```diff
@@ -14,15 +14,14 @@
...
-    | quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-    | consequat.
-  p Odio nostrum nisi vel enim veritatis, facere sunt quo sequi vitae vero suscipit, consequatur cupiditate tenetur maxime perspiciatis neque provident hic ad veniam. Nostrum consequuntur tempore, saepe provident atque accusantium.
+     |Languages used
+     = @project.language
+     br
+     = @project.description
   .git-link
-    a href="" 
+    = link_to "See on Github", url_with_protocol(@project.github_link),  :target => '_blank'
```
Comment (pg20) [inline app/views/mentee_profiles/show.html.slim]: giving error 
undefined method `stringify_keys' for "https://github.com/pg20":String

File: app/views/mentee_profiles/show.html.slim [hunk 7]
```diff
@@ -14,15 +14,14 @@
...
   p
-    | Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-    | tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-    | quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-    | consequat.
-  p Odio nostrum nisi vel enim veritatis, facere sunt quo sequi vitae vero suscipit, consequatur cupiditate tenetur maxime perspiciatis neque provident hic ad veniam. Nostrum consequuntur tempore, saepe provident atque accusantium.
+     |Languages used
+     = @project.language
+     br
+     = @project.description
```
Comment (pg20) [inline app/views/mentee_profiles/show.html.slim]: the project description is a paragraph, if its long, then it should break the paragraph into two and then display it ? or everything is automatically handled :P

**Pull request merged**